community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/RelationshipCountsProcessor.java

```diff
@@ -123,36 +104,63 @@ public boolean process( RelationshipRecord record )
     @Override
     public void done()
     {
-        long index = 0;
-        for ( int startNodeLabelId = 0; startNodeLabelId <= anyLabel; startNodeLabelId++ )
+        for ( int wildcardType = 0; wildcardType <= anyRelationshipType; wildcardType++ )
```

Comment:
I'd simply call this variable `type` instead of `wildcardType`.
IMO it cause confusion with the current name, since it doesn't actually refer to a wildcard (according to how it is used).